test/views/html/incomes/EstimatedPayLandingPageViewSpec.scala

```diff
@@ -29,28 +29,28 @@ class EstimatedPayLandingPageViewSpec extends TaiViewSpec {
   "Estimated Pay Landing Page" should {
     behave like pageWithBackLink
     behave like pageWithCancelLink(Call("GET", controllers.routes.IncomeSourceSummaryController.onPageLoad(empId).url))
-    behave like pageWithCombinedHeaderNewTemplate(
-      messages("tai.howToUpdate.preHeading", employerName),
```

Comment:
@masahmad will this be fine or will it fail the accessibility tests? The pre heading has been removed